src/plugins/vis_type_tagcloud/public/components/tag_cloud_visualization.test.js

```diff
@@ -17,35 +17,24 @@
  * under the License.
  */
 
-import expect from '@kbn/expect';
-import ngMock from 'ng_mock';
-import { ImageComparator } from 'test_utils/image_comparator';
-import basicdrawPng from './basicdraw.png';
-import afterresizePng from './afterresize.png';
-import afterparamChange from './afterparamchange.png';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { ExprVis } from '../../../../../../plugins/visualizations/public/expressions/vis';
+import 'jest-canvas-mock';
+
+import { ExprVis } from '../../../visualizations/public/expressions/vis';
 
 // Replace with mock when converting to jest tests
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { seedColors } from '../../../../../../plugins/charts/public/services/colors/seed_colors';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { BaseVisType } from '../../../../../../plugins/visualizations/public/vis_types/base_vis_type';
-// Will be replaced with new path when tests are moved
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { createTagCloudVisTypeDefinition } from '../../../../../../plugins/vis_type_tagcloud/public/tag_cloud_type';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { createTagCloudVisualization } from '../../../../../../plugins/vis_type_tagcloud/public/components/tag_cloud_visualization';
-import { npStart } from 'ui/new_platform';
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
-import { setFormatService } from '../../../../../../plugins/vis_type_tagcloud/public/services';
-
-const THRESHOLD = 0.65;
-const PIXEL_DIFF = 64;
-describe('TagCloudVisualizationTest', function () {
+import { seedColors } from '../../../charts/public/services/colors/seed_colors';
+import { BaseVisType } from '../../../visualizations/public/vis_types/base_vis_type';
```

Comment:
`BaseVisType` is not actually doing much, check out the implementation here: `src/plugins/visualizations/public/vis_types/base_vis_type.ts`. We can just but an element like this together inside the test, no need to deeply import dependencies.